### DISCUSSION TITLE
Request support for markdown and allow export to .md!!!

### DISCUSSION THREAD

**Discussion by ghost** [zim-desktop-wiki/zim-desktop-wiki#1692] [Ideas]
Using Zim to develop a local wiki note-system is very easy and efficient, the developers are great! It is required to support markdown and allow exporting to md file format.

**Comment by introt** [comment]
Glad you enjoy Zim! Exporting to markdown is already supported, and there's an open issue for supporting markdown as an editable format: #26. See also @pfalcon's fork with markdown support: https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/26#issuecomment-770992975

**Reply by introt** [reply]
See the manual for more info on available export formats: https://zim-wiki.org/manual/Help/Export.html

**Reply by ghost** [reply]
Thanks a lot！Is it possible to open an existing md document and edit it directly?

**Reply by introt** [reply]
You're welcome! It's not possible to edit an existing md document with regular Zim, but that fork should do it. You may also use eg. https://github.com/Xunius/markdown2zim to convert your existing markdown files to Zim Wiki format.

**Reply by grossvater** [reply]
There is any plan to merge back the support for markdown that @falcons's implemented?

**Reply by introt** [reply]
@grossvater please see https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/26#issuecomment-1586076751

**Comment by Mike221223** [comment]
I've seen here and there Jaap mention plans to transition to markdown syntax, and I wonder if it will nullify things possible in Zim and not possible in markdown and vice versa -- things like highlighting, which I find very useful and use regularly; new lines without "new-line" tags required in markdown, etc.

I don't see much point in switching to markdown. In my experience, it's not as "solid" and universally supported as it's made seem by popular opinion. Different markdown editors and platforms recognize markdown differently -- some of them don't recognize certain things. WriteFreely, for instance, couldn't recognize a 'new line', that is two spaces at the end of a line; I had to change it to `</nl>`.

Zim syntax, to me, works perfectly. I also find it much more intuitive than markdown: a 'new line' is a new line -- started by hitting `Enter`; `Tab` is tab -- not 'code', etc. Also, markdown is already possible to an extent with `Insert >> Code block`.

**Reply by rpnfan** [reply]
I think what would solve much already is pasting/ importing md content, which then can be saved as a Zim file. But nowadays you often have md content, which you would like to use in Zim, but cannot without manual re-formatting.
